src/Momento.Sdk/Config/Configuration.cs

```diff
@@ -33,26 +52,51 @@ public Configuration(IRetryStrategy retryStrategy, IList<IMiddleware> middleware
         this.TransportStrategy = transportStrategy;
     }
 
+    /// <summary>
+    ///  Configures logging
```

Comment:
This is technically a copy constructor so we should probably allude to that.

"Creates a new instance of the Configuration object, updated to use the specified logger factory."